charts/nodejs/templates/ingress.yaml

```diff
@@ -55,10 +55,8 @@ spec:
             port:
               number: {{ .Values.service.externalPort }}
 {{- if .Values.service.additionalHosts }}
-{{- range $host := .Values.service.additionalHosts }}
-  - host: {{ $host }}
```

Comment:
Would it be better to rename this to `additionalHost` now that you remove `{{-range }}` and there can only be a single domain name?